packages/clarity-js/src/data/metadata.ts

```diff
@@ -17,6 +17,9 @@ let rootDomain = null;
 export function start(): void {
   rootDomain = null;
   const ua = navigator && "userAgent" in navigator ? navigator.userAgent : Constant.Empty;
+  const timezone = Intl?.DateTimeFormat().resolvedOptions().timeZone ?? '';
```

Comment:
>Intl?.DateTimeFormat().resolvedOptions().timeZone [](http://example.com/codeflow?start=19&length=49)

does look like there are browsers which won't support all of this:

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#browser_compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#browser_compatibility)

should do something safer like Intl?.DateTimeFormat?.()?.resolvedOptions?.()?.timeZone

Reply:
ResolvedOptions is actually supported by all browsers, I added the extra safe validations anyways.